MediaBrowser.Model/Drawing/DrawingUtils.cs

```diff
@@ -57,6 +57,54 @@ public static class DrawingUtils
             return new ImageDimensions(newWidth, newHeight);
         }
 
+        /// <summary>
+        /// Resizes to fill box.
+        /// Returns original size if both width and height are null or zero.
+        /// </summary>
+        /// <param name="size">The original size object.</param>
+        /// <param name="fillWidth">A new fixed width, if desired.</param>
+        /// <param name="fillHeight">A new fixed height, if desired.</param>
+        /// <returns>A new size object or size.</returns>
+        public static ImageDimensions ResizeFill(
+            ImageDimensions size,
+            int? fillWidth,
+            int? fillHeight)
+        {
+            // Return original size if input is invalid.
+            if (
+                (fillWidth == null && fillHeight == null)
+                || (fillWidth == 0 || fillHeight == 0))
+            {
+                return size;
+            }
+
+            if (fillWidth == null || fillWidth == 0)
```

Comment:
This is never true, is it?

Reply:
I see that my logic in the block above is not entirely correct, but this one is true whenever height is specified but not width.